cartoframes/data/utils.py

```diff
@@ -155,3 +156,13 @@ def get_columns(context, query):
 def setting_value_exception(prop, value):
     return CartoException(("Error setting {prop}. You must use the `update` method: "
                            "dataset_info.update({prop}='{value}')").format(prop=prop, value=value))
+
+
+def get_public_carto_context(context):
```

Comment:
Why not `get_public_context`?

Reply:
Because it is returning a CartoContext class.

Reply:
But now Context == CartoContext, right?

Reply:
Until now yes, but we will add more context types in the future. I am going to change it and we will see